addon/locales/en/translations.js

```diff
@@ -223,18 +223,24 @@ export default {
                 chooseProject: 'Choose project',
-                projectSelectMessage: 'The list of projects appearing in the selector are projects and components for which you have admin access. Registrations are not included here.',
+                projectSelectMessage: 'The list of projects appearing are projects and components for which you have write access. Registrations are not included here.',
                 noProjectsExistError: 'You have no available projects. Go back to create a new project.',
                 organizeProject: 'Organize',
-                organizeParentNode: 'You can organize your file by storing it in the project or in its own component. If you select "Make a new component", the file will be stored in a new component inside the project. If you select "Use the current component", the file will be stored inside the project. If you are unsure, select "Make a new component".',
-                organizeChildNode: 'You can organize your file by storing it in this component or in its own component. If you select "Make a new component", the file will be stored in a new component inside this component. If you select "Use the current component", the file will be stored inside this component. If you are unusure, select "Make a new component".',
+                organizeParentNode: 'You can organize your file by storing it in the project or its own component.',
+                organizeChildNode: 'You can organize your file by storing it in the component or its own component.',
                 createComponent: 'Create component',
                 couldNotCreateComponent: 'Could not create component. Please try again.',
                 couldNotCreateProject: 'Could not create project. Please try again.',
                 useCurrentComponent: 'Use current component',
                 useCurrentProject: 'Use current project',
-                convertOrCopyMessage: 'You have selected and organized your file. Clicking "Move file" will immediately make changes to your OSF project and move your file. You can continue viewing and editing your file from your project.',
-                noLongerPublicWarning: 'Files moved to private projects/components will no longer be public or discoverable by others.',
+                convertOrCopyMessage: {
+                    project: 'Clicking "Move file" will immediately make changes to your OSF project and move your file.',
+                    component: 'Clicking "Move file" will immediately make changes to your OSF component and move your file.',
+                },
+                noLongerPublicWarning: {
+                    project: 'Files moved to private projects will no longer be public or discoverable by others.',
+                    component: 'Files moved to private components will no longer be public or discoverable by others.'
```

Comment:
Are quickfiles discoverable/searchable to others? This seems like another thing that was true for preprints but isn't true for quickfiles.

Reply:
They are, because you can either use the API or go to the user's profile page to see their quickfiles.

Reply:
Ah okay, that makes sense.